ONSdigital/ssdc-rm-support-tool
# Add action rule description

# Motivation and Context


We wanted to add a human readable description for the action rules.

# What has changed




Updated the Collection Exercise page to display Action Rule description.
Also updated the New Action rule modal window with the new text field and validation. 

Controversial: added a separate file for the related constants 

# How to test?




Run with docker-dev make sure it validates as expected and displays the description in the table.

# Links



https://jira.ons.gov.uk/secure/RapidBoard.jspa?rapidView=2913&view=detail&selectedIssue=SDCSRM-504

# Screenshots (if appropriate):
